pom.xml

```diff
@@ -119,42 +124,38 @@
          <id>vaadin-addons</id>
          <url>http://maven.vaadin.com/vaadin-addons</url>
       </repository>
+      <repository>
+         <id>spring-milestones</id>
+         <name>Spring Milestones</name>
+         <url>https://repo.spring.io/milestone</url>
+         <snapshots>
+            <enabled>false</enabled>
+         </snapshots>
+      </repository>
    </repositories>
 
    <properties>
+      <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+      <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
       <java.version>1.8</java.version>
-      <spring.boot.version>1.4.7.RELEASE</spring.boot.version>
-      <spring.cloud.version>Camden.SR6</spring.cloud.version>
+      <spring.boot.version>2.1.1.RELEASE</spring.boot.version>
+      <spring.cloud.version>Greenwich.RC2</spring.cloud.version>
+
       <snapshotDependencyAllowed>true</snapshotDependencyAllowed>
 
-      <!-- Spring boot version overrides (should be reviewed with every boot upgrade) - START -->
+      <!-- Spring boot version overrides (should be reviewed with every boot 
+         upgrade) - START -->
       <!-- Newer versions needed than defined in Boot -->
-      <!-- Deserializer security vulnerability fixed -->
-      <jackson.version>2.8.9</jackson.version>
-      <!-- Performance fixes -->
-      <spring-hateoas.version>0.23.0.RELEASE</spring-hateoas.version>
-      <!-- Java 8 support -->
-      <assertj.version>3.6.2</assertj.version>
-      <!-- Repeatable migrations, Disabling clean etc. -->
-      <flyway.version>4.0.3</flyway.version>
-      <!-- Improved integration test capabilities -->
-      <spring-amqp.version>1.7.6.RELEASE</spring-amqp.version>
-      <!-- Compatability with RabbitMQ >= 3.7.3 -->
-      <rabbitmq.http-client.version>1.3.1.RELEASE</rabbitmq.http-client.version>
+      <!-- Fixed dependencies -->
+      <rabbitmq.http-client.version>2.1.0.RELEASE</rabbitmq.http-client.version>
```

Comment:
We need to upgrade to at least Flyway 5.2.4 to avoid https://github.com/flyway/flyway/issues/2215
Thanks

Reply:
`<flyway.version>5.2.4</flyway.version>`

Reply:
Will be fixed with Boot 2.1.2 . I will upgrade soon.